modules/graphql/src/endpoints.rs

```diff
@@ -15,8 +13,7 @@ async fn index_graphiql() -> Result<HttpResponse> {
 
 pub fn configure(svc: &mut web::ServiceConfig, db: Database) {
     let schema = Schema::build(RootQuery::default(), EmptyMutation, EmptySubscription)
-        .data::<Arc<Graph>>(Arc::new(Graph::new(db.clone())))
-        .data::<Arc<Database>>(Arc::new(db))
+        .data::<Database>(db)
```

Comment:
Does this need to be wrapped back in an Arc<Database>

Reply:
@bobmcwhirter,  I think it does, restoring...

Reply:
@bobmcwhirter , That solves all context issues !